### PR TITLE
apx: refresh page

### PR DESCRIPTION
**Pull request by kbdharun**
## Changes

- Updated apx page.
- Updated Tamil translation of apx page.

Signed-off-by: K.B.Dharun Krishna <kbdharunkrishna@gmail.com>

- [x] The page(s) are in the correct platform directories: `linux`.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

### CODE REVIEW

File: pages/linux/apx.md [hunk 1]
```diff
@@ -1,17 +1,21 @@
...
 
 > Package management utility for Vanilla OS.
-> Install packages inside managed containers from multiple sources (`apx` supports --aur,--dnf flags in all commands).
+> Install packages inside managed containers from multiple sources (`apx` supports --aur,--dnf, --apk flags in all commands).
 > More information: <https://github.com/Vanilla-OS/apx>.
 
-- Initialize the container:
+- Initialize/Reinitialize the container:
 
 `apx init`
```
Comment (EmilySeville7cfg) [inline pages/linux/apx.md:9]:
```suggestion
- Initialize or reinitialize the current container:

`apx init`
```

Optional fix: unexpected `/` delimiter, `or` expected.
Optional fix: unexpected `container` noun, `current` expected.

File: pages/linux/apx.md [hunk 2]
```diff
@@ -1,17 +1,21 @@
...
 
 - Install specific packages in the container:
 
 `apx install {{package1 package2 ...}}`
 
+- Install a DEB/RPM package inside the container (Use `--dnf` flag for installing RPMs):
+
+`apx install --sideload {{path/to/package{{.deb|.rpm}}}}`
```
Comment (EmilySeville7cfg) [inline pages/linux/apx.md:17]:
```suggestion
- Install a DEB/RPM package inside the container (Use `--dnf` flag for installing RPMs):

`apx install --sideload {{path/to/package.ext}}`
```

Mandatory fix: unexpected `path/to/package{{.deb|.rpm}}` placeholder content, `ext` expected.
Note: nested placeholders are not supported in specification.

**Review by EmilySeville7cfg — CHANGES_REQUESTED** [review]
Mandatory fixes found, changes requested.
Thanks for a contribution! 🚀

File: pages/linux/apx.md (hunk 1 above)
Comment (kbdharun) [reply pages/linux/apx.md:9]: `the current` won't be apt here, I will instead use a specific container as this command supports flags too.

**Review by EmilySeville7cfg — APPROVED** [review]

**Review by blueskyson — APPROVED** [review]

**Review by mfrw — APPROVED** [review]
LGTM for english :)

**Pull request merged**